draft-ietf-quic-multipath.md

```diff
@@ -381,6 +381,12 @@ that address. After receiving packets from the
 client on a new path, if the server decides to use the new path,
 the server MUST perform path validation ({{Section 8.2 of QUIC-TRANSPORT}})
 unless it has previously validated that address.
+A server can also decide not to perform path validation if it wants to deny the new path,
+or not perform it immediately if, e.g., no connection ID is yet available.
+If an endpoint decides to deny the establishment of a new path,
+this should be considered as a path closure and therefore requires sending a
+PATH_ABANDON frame on another path for the associated Path ID used
+by the client as specified in {{path-close}}.
```

Comment:
An endpoint that receives a PATH_CHALLENGE and does not want to establish this path should immediately close the path by sending a PATH_ABANDON on another path, as specified in section {{path-close}}.

An endpoint that lacks resource to immediately configure a new path MAY delay sending the PATH_RESPONSE until sufficient resource are available. Long delays may cause the peer to repeat the PATH_CHALLENGE and eventually send a PATH_ABANDON, in which case the procedures specified in  section {{path-close}} apply.


If an endpoint decides to deny the establishment of a new path,
it may not response to the path challenge. However,
this should be considered as a path closure and therefore requires sending a
PATH_ABANDON frame on another path for the associated Path ID used
by the client as specified in {{path-close}}.

Reply:
I have committed your first sentence. I will create a separate PR for you section sentence. The first part has been discussed and can be directly merged. So the second we might want to open an issue and way a bit for further feedback just to ensure transparency.

Reply:
Two nits: 

1) the "should" in the first sentence SHOULD be capitalized.
2) the second semtence is the reason it I wrote "SHOULD" and not "MUST", so the two sentences complement each other.

Reply:
I don't think the second sentence is the reason for a SHOULD. In the first sentence the server doesn't want to open the path. In the second sentence the server wants to open the path but can't. So I think if the server does NOT want to open the path, it MUST send the path_abandon.

However, I don't think we should use normative language in this sentence at all because the normative part is the closing section which says:
> An endpoint that wants to close a path MUST explicitly
> terminate the path by sending a PATH_ABANDON frame.

This applies to all situation where an endpoint wants to close a path. We don't need to specify the same thing normatively twice.